README.md

```diff
@@ -129,8 +129,8 @@ sensors:
   - id: cpu_package
     # The type of sensor configuration, one of: hwmon | file
     hwmon:
-      # The controller platform as displayed by `fan2go detect`, f.ex.:
-      # "nouveau", "coretemp" or "it8620" etc.
+      # A regex matching a controller platform displayed by `fan2go detect`, f.ex.:
+      # "coretemp", "it8620", "corsaircpro-*" etc.
```

Comment:
Thinking about it, this should be `.*` at the end, right? So:
`corsaircpro-.*`

Or can `regexp.MatchString` deal with the `*` at the end?

Reply:
`regexp.MatchString` can deal with the `*` at the end. When matching I believe its matching for the regexp inside of the string which is fine enough for our purposes. I just ran a quick test using my own config.